h2o-r/tests/runitUtils/utilsR.R

```diff
@@ -931,27 +931,33 @@ random_NN <- function(actFunc, max_layers, max_node_number) {
 # Parameters:  frame1, frame2: H2O frames to be compared.
 #              tolerance: tolerance of comparison
 #----------------------------------------------------------------------
-compareFrames <- function(frame1, frame2, prob=0.5, tolerance=1e-6) {
+compareFrames <- function(frame1, frame2, prob=0.5, tolerance=1e-6, enum2String=FALSE) {
```

Comment:
`enum2Strings` - is the purpose of the flag to ignore the order of the factors? can we find a better fitting name if that is the case? eg. ignore_factor_order?